### DISCUSSION TITLE
Ugly representation of editable block in backend.

### DISCUSSION THREAD

**Discussion by sandra-walker** [pimcore/pimcore#8811] [Q&A / Support] [answered]
Hello boys and girls...

I have a silly BS4 container, row, and some cols with BS card. nothing special.
Want to put this in a pimcore block. tecnicaly it works perfect. But in backend it is displayd very ugly.
is there a way to handle this?
![pc-backend](https://user-images.githubusercontent.com/37797118/114866368-c012bf80-9df3-11eb-8417-0aaeb355fce4.PNG)

**Comment by passioneight** [comment, marked as answer]
You could always use the `editmode` variable to render the template differently:

```twig
{% if editmode %}
    {# TODO: render template for editmode #}
{% else %}
    {# TODO: render template for frontend #}
{% endif %}
```

Also, checkout the [manual mode for blocks](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Block.html#page_Using-Manual-Mode) if you need even more customization.
